src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map.Entry;
+
+public class LottoResult {
+
+    public static final int LOTTO_PRICE = 1000;
+    private static final int DECIMAL_TRIM_NUMERATOR = 100;
+    private static final double DECIMAL_TRIM_DENOMINATOR = 100.00;
+
+    private final EnumMap<LottoRank, Integer> lottoResultStatistics;
```

Comment:
```suggestion
    private final Map<LottoRank, Integer> lottoResultStatistics;
```

두 코드는 어떠한 차이가 있을지 고민해보아요